### DISCUSSION TITLE
Are DM Channel permissions broken?

### DISCUSSION THREAD

**Discussion by shivangrathore** [Rapptz/discord.py#7021] [Q&A] [answered]
```py
DMChannel.permissions_for(bot.user).read_messages 
# Returns false everytime, even if bot is able to read messages
```

**Comment by Rapptz** [comment, marked as answer]
This is indeed a bug. Good catch. This will be fixed in v2.0 ([commit](https://github.com/Rapptz/discord.py/commit/27556ea0a28ae58bc3319e2209e70042f4fb0213)). I might backport it to v1.7.x if more bugs come up.
